app/executor.php

```diff
@@ -283,8 +283,8 @@ function getStorageDevice($root): Device {
             $endTime = \time();
             $container = array_merge($container, [
                 'status' => 'ready',
-                'stdout' => \utf8_encode($stdout),
-                'stderr' => \utf8_encode($stderr),
+                'stdout' => \mb_strcut(\utf8_encode($stdout), 0, 1000000), // Limit to 1MB
+                'stderr' => \mb_strcut(\utf8_encode($stderr), 0, 1000000), // Limit to 1MB
```

Comment:
We keep utf8_encode here, but we remove it from elsewhere. Is that fine?

Reply:
Thank you for bringing this up, It is now fixed. 👍